### DISCUSSION TITLE
Option to invert fan controls

### DISCUSSION THREAD

**Discussion by Viktska** [Rem0o/FanControl.Releases#1448] [Ideas]
Suggestion to add "Inverted speed" option in fan card hamburger menu. 

Example why it might be needed.
![image](https://user-images.githubusercontent.com/122309910/211386862-370610f5-843d-4b9e-baeb-e965e52568f0.png)
![image](https://user-images.githubusercontent.com/122309910/211387926-979cacd1-78da-4718-b5e8-8fe2ee8711a0.png)
![image](https://user-images.githubusercontent.com/122309910/211388757-78c3737a-0246-4878-afd0-3f9cd0e755d3.png)

**Comment by Huddling1705** [comment]
I know it would look wierd, but I guess there's nothing preventing you from starting your curve at 100% and ending it at 0%

**Reply by Rem0o** [reply]
The fact your control mapping is reversed is reallllly weird to me. What's your superIO chip and motherboard? (Copy report to clipboard), please don't paste the whole thing.
